index.html

```diff
@@ -3250,6 +3250,70 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="normative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of <a>properties</a> to serve as possible
+extension points. While some implementers signaled interest in these properties,
+their inclusion in this specification was considered to be premature; these
+extension points might be more formally defined in future versions of this
+specification. It is important to note that these properties are not defined by
+this specification and implementers are cautioned that usage of these properties
+is considered experimental.
+        </p>
+        <p>
+Implementers MAY use these properties, but SHOULD expect them and/or
+their meanings to change during the process to normatively specify them.
+Implementers SHOULD NOT use these properties without a publicly disclosed
+specification describing their implementation.
+        </p>
+
+        <p>
+In order to avoid collisions regarding how the following properties are used,
+implementations MUST specify a `type` property in the value associated with the
+reserved property. For more information related to adding `type` information,
+see Section <a href="#types"></a>.
+        </p>
+
+        <p class="issue" title="Reserved properties under debate">
+The following extension point properties are under consideration for being
+marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
+`evidence`, `renderMethod`, `refreshService`, and
+`confirmationMethod/confidenceMethod`.
+        </p>
```

Comment:
I'd remove the concrete reserved properties for now and only merge the PR with an empty list. A first stab on the list can then follow. Otherwise looks good to me.

Reply:
In the previous conversations we've held there has been wide agreement that this PR should not contain any entries in the table, so it doesn't.
 PR #1097 has been raised to introduce them.